src/components/header/PageHeader.jsx

```diff
@@ -98,46 +122,75 @@ class PageHeader extends Component {
     </div>
   )
 
-  render() {
+  renderFilterStatus = () => {
     const {
       selectedStatus,
-      statuses,
       showFilterOptionsDropdown,
       activeClass,
     } = this.state;
+    const { statuses } = this.props;
+    return (
+      <div className='filterOptions'>
+        <button
+          className='filterOptions__button'
+          onClick={this.createFilterOptionsButtonClickHandler()}
+        >
+          {capitalizeString(selectedStatus)}
+        </button>
+        <div className={this.getDropdownClassName(
+          showFilterOptionsDropdown,
+          ['filterOptions__dropdown'],
+        )}
+        >
+          {
+            statuses.map(status => (
+              <div
+                key={status}
+                onMouseDown={(e) => {
+                  if (this.props.changeFilterHandler()) {
+                    this.props.filterRedemptions(e, status);
+                  } else {
+                    this.props.filterActivities(e.currentTarget.textContent);
+                  }
+                }}
+                className={`filterOptions__option ${selectedStatus === status ? activeClass : ''}`}
+                role='button'
+                tabIndex='0'
+              >
+                {status && capitalizeString(status)}
+              </div>
+            ))
+          }
+        </div>
+      </div>
+    );
+  }
+
+  render() {
+    const {
+      tabs,
+      title,
+      selectedSociety,
+      showTabs,
+    } = this.props;
     return (
       <header className='pageHeader'>
-        <h1 className='pageTitle'>{this.props.title}</h1>
-        {this.props.showSelectAllApproveBtn && this.renderSelectAllApprovebtn()}
+        {
+          showTabs ?
+            <Tabs
+              tabTitles={tabs}
+              handleChangeTab={this.props.handleChangeTab}
+              selectedTab={selectedSociety}
+            />
```

Comment:
Same as `iconHtml` comment. This would look much better in terms of being able to read.